Docs build fails due to conda netcdf issue

The documentation builds locally, but fails in the CI (same problem occurs over on RTD):

https://github.com/CWorthy-ocean/roms-tools/actions/runs/11370740902/job/31632008850?pr=165

```
Collecting netcdf4 (from roms-tools==0.1.dev1)
  Downloading netcdf4-1.7.1.post2.tar.gz (827 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 827.8/827.8 kB 77.3 MB/s eta 0:00:00
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'error'
  error: subprocess-exited-with-error
  
  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [45 lines of output]
      Package hdf5 was not found in the pkg-config search path.
      Perhaps you should add the directory containing `hdf5.pc'
      to the PKG_CONFIG_PATH environment variable
      Package 'hdf5', required by 'virtual:world', not found
      reading from setup.cfg...
          HDF5_DIR environment variable not set, checking some standard locations ..
      checking /home/runner/micromamba/envs/romstools/include...
      hdf5 headers not found in /home/runner/micromamba/envs/romstools/include
      checking /home/runner/micromamba/envs/romstools/Library/include...
      hdf5 headers not found in /home/runner/micromamba/envs/romstools/Library/include
      checking /home/runner/include...
      hdf5 headers not found in /home/runner/include
      checking /usr/local/include...
      hdf5 headers not found in /usr/local/include
      checking /sw/include...
      hdf5 headers not found in /sw/include
      checking /opt/include...
      hdf5 headers not found in /opt/include
      checking /opt/local/include...
      hdf5 headers not found in /opt/local/include
      checking /opt/homebrew/include...
      hdf5 headers not found in /opt/homebrew/include
      checking /usr/include...
      hdf5 headers not found in /usr/include
      Traceback (most recent call last):
        File "/home/runner/micromamba/envs/romstools/lib/python3.13/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line [35](https://github.com/CWorthy-ocean/roms-tools/actions/runs/11370740900/job/31631201289?pr=165#step:4:36)3, in <module>
          main()
          ~~~~^^
        File "/home/runner/micromamba/envs/romstools/lib/python3.13/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 335, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
                                   ~~~~^^^^^^^^^^^^^^^^^^^^^^^^
        File "/home/runner/micromamba/envs/romstools/lib/python3.13/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 118, in get_requires_for_build_wheel
          return hook(config_settings)
        File "/tmp/pip-build-env-e118lk_0/overlay/lib/python3.13/site-packages/setuptools/build_meta.py", line 332, in get_requires_for_build_wheel
          return self._get_build_requires(config_settings, requirements=[])
                 ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/tmp/pip-build-env-e118lk_0/overlay/lib/python3.13/site-packages/setuptools/build_meta.py", line 302, in _get_build_requires
          self.run_setup()
          ~~~~~~~~~~~~~~^^
        File "/tmp/pip-build-env-e118lk_0/overlay/lib/python3.13/site-packages/setuptools/build_meta.py", line 318, in run_setup
          exec(code, locals())
          ~~~~^^^^^^^^^^^^^^^^
        File "<string>", line 277, in <module>
        File "<string>", line 226, in _populate_hdf5_info
      ValueError: did not find HDF5 headers
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
Error: Process completed with exit code 1.
```

Strangely, installing the netcdf dependency in the other CI jobs with conda works perfectly.

--- a/.github/workflows/sphinx.yaml
+++ b/.github/workflows/sphinx.yaml
@@ -17,8 +17,8 @@ jobs:
           cache-environment: true
           micromamba-version: 'latest'
           environment-file: ci/environment.yml
-          create-args: |
-            python=${{ matrix.python-version }}
+      - name: Conda info
+        run: conda info
       - name: Install roms-tools
         shell: micromamba-shell {0}
         run: |

--- a/.github/workflows/sphinx.yaml
+++ b/.github/workflows/sphinx.yaml
@@ -20,7 +20,7 @@ jobs:
           cache-downloads: true
           cache-environment: true
           micromamba-version: 'latest'
-          environment-file: ci/environment.yml
+          environment-file: docs/environment.yml
           create-args: |
             python=${{ matrix.python-version }}
           cache-key: micromamba-${{ matrix.python-version }}-${{ runner.os }}

--- a/.github/workflows/sphinx.yaml
+++ b/.github/workflows/sphinx.yaml
@@ -7,7 +7,11 @@ on:
 
 jobs:
   docs:
+    name: ${{ matrix.python-version }}-build-without-dask
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.12"]
     steps:
       - uses: actions/checkout@v4
       - name: Create conda environment
@@ -17,6 +21,9 @@ jobs:
           cache-environment: true
           micromamba-version: 'latest'
           environment-file: ci/environment.yml
+          create-args: |
+            python=${{ matrix.python-version }}
+          cache-key: micromamba-${{ matrix.python-version }}-${{ runner.os }}
       - name: Conda info
         run: conda info
       - name: Install roms-tools

--- a/.github/workflows/sphinx.yaml
+++ b/.github/workflows/sphinx.yaml
@@ -7,7 +7,7 @@ on:
 
 jobs:
   docs:
-    name: ${{ matrix.python-version }}-build-without-dask
+    name: ${{ matrix.python-version }}-build-docs
     runs-on: ubuntu-latest
     strategy:
       matrix:

--- a/.github/workflows/sphinx.yaml
+++ b/.github/workflows/sphinx.yaml
@@ -7,11 +7,7 @@ on:
 
 jobs:
   docs:
-    name: ${{ matrix.python-version }}-build-docs
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python-version: ["3.12"]
     steps:
       - uses: actions/checkout@v4
       - name: Create conda environment
@@ -21,9 +17,6 @@ jobs:
           cache-environment: true
           micromamba-version: 'latest'
           environment-file: docs/environment.yml
-          create-args: |
-            python=${{ matrix.python-version }}
-          cache-key: micromamba-${{ matrix.python-version }}-${{ runner.os }}
       - name: Conda info
         run: conda info
       - name: Install roms-tools

--- a/.readthedocs.yaml
+++ b/.readthedocs.yaml
@@ -16,7 +16,7 @@ sphinx:
    configuration: docs/conf.py
 
 conda:
-    environment: ci/environment.yml
+    environment: docs/environment.yml
 
 # Python dependencies and install method
 python:

--- a/ci/environment.yml
+++ b/ci/environment.yml
@@ -1,4 +1,4 @@
-name: romstools
+name: romstools-test
 channels:
   - conda-forge
   - nodefaults
@@ -12,10 +12,3 @@ dependencies:
   - black
   - pre-commit
   - coverage
-  # docs
-  - sphinx
-  - nbsphinx
-  - sphinx-book-theme
-  - sphinxcontrib-bibtex
-  # for running jupyter notebooks
-  - ipykernel

--- a/docs/contributing.rst
+++ b/docs/contributing.rst
@@ -49,4 +49,3 @@ Then navigate to the docs folder and build the docs via::
     make html
 
 You can now open ``docs/_build/html/index.html`` in a web browser.
-

--- a/docs/contributing.rst
+++ b/docs/contributing.rst
@@ -1,0 +1,52 @@
+Contributor Guide
+##################
+
+Running the tests
+=================
+Install and activate the following conda environment::
+
+    cd roms-tools
+    conda env create -f ci/environment.yml
+    conda activate romstools-test
+    pip install -e ".[dask]"
+
+You can check the functionality of the ROMS-Tools code by running the test suite::
+
+    conda activate romstools-test
+    pytest
+
+
+Contributing code
+=================
+
+If you have written new code, you can run the tests as described in the previous step. You will likely have to iterate here several times until all tests pass.
+The next step is to make sure that the code is formatted properly. Activate the environment::
+
+    conda activate romstools-test
+
+You can now run all the linters with::
+
+    pre-commit run --all-files
+
+Some things will automatically be reformatted, others need manual fixes. Follow the instructions in the terminal until all checks pass.
+Once you got everything to pass, you can stage and commit your changes and push them to the remote github repository.
+
+
+Building the documentation locally
+==================================
+
+Install and activate the following conda environment::
+
+    cd roms-tools
+    conda env create -f docs/environment.yml
+    conda activate romstools-docs
+    pip install -e ".[dask]"
+
+Then navigate to the docs folder and build the docs via::
+
+    cd docs
+    make fresh
+    make html
+
+You can now open ``docs/_build/html/index.html`` in a web browser.
+

--- a/docs/contributing.rst
+++ b/docs/contributing.rst
@@ -42,9 +42,10 @@ Install and activate the following conda environment::
     conda activate romstools-docs
     pip install -e ".[dask]"
 
-Then navigate to the docs folder and build the docs via::
+Navigate to the docs folder and build the docs with the activated environment::
 
     cd docs
+    conda activate romstools-docs
     make fresh
     make html
 

--- a/docs/environment.yml
+++ b/docs/environment.yml
@@ -1,0 +1,12 @@
+name: romstools-docs
+channels:
+  - conda-forge
+  - nodefaults
+dependencies:
+  - python=3.12
+  - sphinx
+  - nbsphinx
+  - sphinx-book-theme
+  - sphinxcontrib-bibtex
+  # for running jupyter notebooks
+  - ipykernel

--- a/docs/index.rst
+++ b/docs/index.rst
@@ -55,9 +55,9 @@ This Python package is inspired by the `UCLA MATLAB tools <https://github.com/nm
 
 .. toctree::
    :maxdepth: 1
-   :caption: Community
+   :caption: Contributor Guide
 
-   support
+   contributing
 
 .. toctree::
    :maxdepth: 1

--- a/docs/installation.rst
+++ b/docs/installation.rst
@@ -30,4 +30,3 @@ If you want to use ROMS-Tools together with dask (which we recommend), you can
 install ROMS-Tools along with the additional dependency via::
 
     pip install ".[dask]"
-

--- a/docs/installation.rst
+++ b/docs/installation.rst
@@ -22,57 +22,12 @@ To obtain the latest development version, first clone
     git clone https://github.com/CWorthy-ocean/roms-tools.git
     cd roms-tools
 
-Next, install and activate the following conda environment::
+Then, install ROMS-Tools via::
 
-    conda env create -f ci/environment.yml
-    conda activate romstools
-
-Finally, install ROMS-Tools in the same environment::
-
-    pip install -e .
+    pip install .
 
 If you want to use ROMS-Tools together with dask (which we recommend), you can
 install ROMS-Tools along with the additional dependency via::
 
-    pip install -e .[dask]
+    pip install ".[dask]"
 
-Running the tests
-=================
-
-You can check the functionality of the ROMS-Tools code by running the test suite::
-
-    conda activate romstools
-    cd roms-tools
-    pytest
-
-
-Contributing code
-=================
-
-If you have written new code, you can run the tests as described in the previous step. You will likely have to iterate here several times until all tests pass.
-The next step is to make sure that the code is formatted properly. Activate the environment::
-
-    conda activate romstools
-
-You can now run all the linters with::
-
-    pre-commit run --all-files
-
-Some things will automatically be reformatted, others need manual fixes. Follow the instructions in the terminal until all checks pass.
-Once you got everything to pass, you can stage and commit your changes and push them to the remote github repository.
-
-
-Building the documentation locally
-==================================
-
-Activate the environment::
-
-    conda activate romstools
-
-Then navigate to the docs folder and build the docs via::
-
-    cd docs
-    make fresh
-    make html
-
-You can now open ``docs/_build/html/index.html`` in a web browser.

--- a/docs/support.rst
+++ b/docs/support.rst
@@ -1,7 +1,0 @@
-Getting Support
-###############
-
-If you think there is a bug in the code or have any questions about using roms-tools,
-please
-`submit an issue <https://github.com/CWorthy-ocean/roms-tools/issues/new>`_,
-with a sufficient description of the problem or question.

